### PR TITLE
Update Equilibrium adapter

### PR DESCRIPTION
**Pull request by Veniamin**
Updated Equilibrium eosdt adapter to Equilibrium polkadot parachain adapter (polkadot version of Genshiro parachain). Equilibrium need to be added to https://github.com/DefiLlama/sushi-analytics/blob/master/adapterMapping.js before adding to production.

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/equilibrium: 
        
 
  
    
    
    Error: Data for protocol equilibrium on chain equilibrium is undefined on heroku
    Truncated error stack:
    
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/projects/helper/heroku-api.js:11:19
        at async getTvl (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:52:23)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:148:9
        at async Promise.all (index 0)
        at async Promise.all (index 1)
        at async /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/test.js:193:3

**Comment by Veniamin** [comment]
pull request for sushi analytics https://github.com/DefiLlama/sushi-analytics/pull/8

File: projects/equilibrium/api.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	const provider = new WsProvider("wss://node.pol.equilibrium.io/");
+	const api = await ApiPromise.create({ provider });
+
+	const assets = (await api.query.eqAssets.assets()).unwrap();
+
+	const queries = assets.map(({ id }) => {
+		return [api.query.eqAggregates.totalUserGroups, ["Balances", { 0: id }]];
+	});
+
+	const balances = await api.queryMulti(queries);
```
Comment (g1nt0ki) [inline projects/equilibrium/api.js:66]: hi @Veniamin thanks for the PR, when I try to run it, I am getting this error

`RPC-CORE: subscribeStorage(keys?: Vec<StorageKey>): StorageChangeSet:: createType(Vec<StorageKey>):: createType(Lookup8):: Assertion failed

Comment (Veniamin) [reply projects/equilibrium/api.js:66]: Hi, sorry for late response, checking

Comment (Veniamin) [reply projects/equilibrium/api.js:66]: There is an error in @polkadot/api 8.8.2, can we update it to the current version 10.2.1?

**Pull request merged**